Singularity/Singularity/Screen/ScreenClasses/GamePauseScreen.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace Singularity.Screen.ScreenClasses
+{
+    /// <inheritdoc cref="IScreen"/>
+    /// <summary>
+    /// Shows the pause menu when in the middle of a game. It shows the
+    /// following: Resume, Quick Save, Save, Statistics, Quit Game.
+    /// </summary>
+    class GamePauseScreen : IScreen
+    {
+        /// <summary>
+        /// Updates the contents of the screen.
+        /// </summary>
+        /// <param name="gametime">Current gametime. Used for actions
+        /// that take place over time </param>
+        public void Update(GameTime gametime)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Draws the content of this screen.
+        /// </summary>
+        /// <param name="spriteBatch">spriteBatch that this object should draw to.</param>
+        public void Draw(SpriteBatch spriteBatch)
+        {
+            throw new NotImplementedException();
```

Comment:
now we have a pause-screen but it is incapabe of actually pausing the game? o.O

Reply:
I just made all the screens but they have not been implemented yet.

Reply:
could you give them some functionality though? I mean ... ok sure

Reply:
It's outside the scope of create game menu and also, there's no way to bring up the screen yet so i haven't worked on it. I can work on it next sprint though